Just_Dail_Scrapper.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+import urllib
+import csv
+import constant
+
+
+def scraper():
+	def get_name(body):
+		return body.find('span', {'class':constant.nameclass}).a.string
+
+	def get_phone_number(body):
+		try:
+			return body.find('p', {'class':'contact-info'}).span.a.string
+		except AttributeError:
+			return ''
+
+
+	def get_address(body):
+		return body.find('span', {'class':'mrehover'}).text.strip()
+
+#page count for page
+	page_number = 1
+	service_count = 1
+
+
+	fields = ['Name', 'Phone', 'Address']
+	out_file = open('Mumbai_electricians.csv','w')
+	csvwriter = csv.DictWriter(out_file, delimiter=',', fieldnames=fields)
+
+	while True:
+
+
+			if page_number > 10:
+				break
+
+			
+			req = urllib.request.Request(constant.url, headers={'User-Agent' : "Magic Browser"}) 
+			page = urllib.request.urlopen( req )
+
+
+			soup = BeautifulSoup(page.read(), "html.parser")
+			services = soup.find_all('li', {'class': 'cntanr'})
+
+
+			
+			for service_html in services:
+				dict_service = {}
+				name = get_name(service_html)
```

Comment:
Make a single function to extract all details out instead of individual functions